FABulous/fabric_files/FABulous_project_template_verilog/Test/README.md

```diff
@@ -1,7 +1,7 @@
-This assumes the default instructions were followed to build a 8x14 fabric in `../../fabric_generator`.
+This assumes FABulous is installed properly and the default instructions were followed to build default fabric.
 
 Latest Yosys and `nextpnr-generic` from upstream (_not_ the old FABulous nextpnr fork) are used
 to build the test design.
 
-Run `build_test_design.sh` to create the bitstream and `run_simulation.sh` to compare a simulation
+Type `make build_test_design` to create the bitstream and `make run_simulation` to compare a simulation
```

Comment:
Likely everyone will have `make`  installed, but I guess still worth mention to install `make` by `sudo apt-get install make` .